core/templates/dev/head/pages/story-editor-page/navbar/story-editor-navbar.directive.ts

```diff
@@ -101,6 +101,13 @@ angular.module('oppia').directive('storyEditorNavbar', [
                     );
                     break;
                   }
+                  if (summaries[idx].language_code !== 'en') {
```

Comment:
Do you need more validations here to match the rest of the backend logic?

Reply:
Yeah, though I meant this PR to only deal with the backend validations. Should I add these also in this PR?